addons/im_livechat/static/src/core/common/livechat_command_dialog.scss

```diff
@@ -0,0 +1,6 @@
+.o-livechat-LivechatCommandDialog-form {
+    &:focus-within {
+        border-color: $input-focus-border-color;
+        color: $input-focus-color;
+    }
+}
```

Comment:
No need to nest the style, can be `.o-livechat-LivechatCommandDialog-form:focus-within {`